src/DetailsView/details-view-initializer.ts

```diff
@@ -97,221 +98,209 @@ if (isNaN(tabId) === false) {
     chromeAdapter.getTab(
         tabId,
         (tab: Tab): void => {
```

Comment:
All these changes suggest to me that this may be a good time to break apart this function to a named variable ('createDefaultRenderer' potentially?) instead.

Reply:
Agreed that we need a better strategy to initialize the details view page. Don't want to rush it in this PR though.